### PR TITLE
[ClockSync] fix NotImplementedError and attribute error

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_commons/async_job.pxd [hunk 1]
```diff
@@ -20,10 +20,10 @@ cdef class AsyncJob:
     cdef object callback                    # Callable
     cdef object job_task                    # Asyncio.Task
     cdef object job_periodic_task           # Asyncio.Task
-    cdef public object idle_task_event      # Asyncio.Event
+    cdef public object idle_task_event
+    cdef public bint should_stop   # Asyncio.Event
 
     cdef bint is_started
-    cdef bint should_stop
     cdef bint is_periodic
```
Comment (Herklos) [inline octobot_commons/async_job.pxd:27]: ?

Comment (GuillaumeDSM) [reply octobot_commons/async_job.pxd:27]: used in clock synch to know if the job is running

Comment (Herklos) [reply octobot_commons/async_job.pxd:27]: Oh ok it was not in the commit diff.

Comment (GuillaumeDSM) [reply octobot_commons/async_job.pxd:27]: (yes, it was another issue XD)

Comment (Herklos) [reply octobot_commons/async_job.pxd:27]: I think we should create a method like `is_stopping` instead of reading `should_stop` value

**Review by Herklos — APPROVED** [review]
:+1:

**Pull request merged**